keyboards/whirlwind_fx_element/keyboard.json

```
@@ -0,0 +1,304 @@
+{
+    "manufacturer": "WhirlwindFX",
+    "keyboard_name": "Element",
+    "maintainer": "aedanmills",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "build": {
+        "lto": true
+    },
+    "matrix_pins": {
+        "rows": [ "B0", "B1", "B2", "B3", "B12", "B13", "B14", "B15" ],
+        "cols": [ "A0", "A1", "A2", "A3", "A4", "A5", "A6", "A7", "A8", "A9", "A10", "A13", "A14" ]
+    },
+    "processor": "STM32F303",
+    "url": "https://www.whirlwindfx.com/products/element",
+    "usb": {
+        "vid": "0x0483",
+        "pid": "0xA33E",
+        "device_version": "1.0.6"
+    },
+    "indicators": {
+        "caps_lock": "B9",
+        "num_lock": "B8",
+        "scroll_lock": "B10",
+        "on_state": 0
+    },
```

add Community Layout support
```suggestion
    },
    "community_layouts": ["fullsize_ansi"],
```